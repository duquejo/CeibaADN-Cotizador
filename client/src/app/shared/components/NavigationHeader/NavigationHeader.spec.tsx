import * as React from 'react';
import { shallow } from 'enzyme';
import { NavigationHeader } from './index';

describe('Pruebas sobre componentes del header <NavigationHeader />', () => {

    it('DeberÃ­a renderizar el elemento adecuadamente', () => {
        const wrapper = shallow( <NavigationHeader /> );
        expect( wrapper ).toMatchSnapshot();
    }); 
});
