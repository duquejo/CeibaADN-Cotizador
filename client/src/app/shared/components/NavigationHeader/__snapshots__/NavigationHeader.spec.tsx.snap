// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componentes del header <NavigationHeader /> Deber√≠a renderizar el elemento adecuadamente 1`] = `
<styled.nav>
  <NavBrand
    imgSrc="logo-ceiba.png"
    text="Ceiba Software"
  />
  <NavList
    items={
      Array [
        Object {
          "label": "Home",
          "url": "/home",
        },
        Object {
          "label": "Administrador",
          "url": "/administrador",
        },
        Object {
          "label": "Cotizador",
          "url": "/cotizador",
        },
      ]
    }
  />
</styled.nav>
`;
